vars/buildApp.groovy

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/groovy
 
 def call(body) {
+    // Metrics
+    def metricsFilePath = "/usr/local/bin/pay-graphite-metric"
+    def metricsFile = new File(metricsFilePath)
+    def metricsCommand = "/bin/bash $metricsFilePath"
```

Comment:
We can change this to Max's new ruby script when done. I deployed this using the pay-deploy-notifications package